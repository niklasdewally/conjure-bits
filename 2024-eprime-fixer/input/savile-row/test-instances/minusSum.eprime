language ESSENCE' 1.0

find q: int(1..5)

such that

(q-1) = (q-q)
