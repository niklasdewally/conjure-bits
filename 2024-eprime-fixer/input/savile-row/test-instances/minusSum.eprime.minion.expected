MINION 3

#This file has been automatically generated by TAILOR v0.3.3
#bug-reports: andrea@cs.st-and.ac.uk

**VARIABLES**
DISCRETE q {1..5}

# aliases

**SEARCH**

PRINT [
[q]]

VARORDER [
q]


**CONSTRAINTS**

eq(q, 1)

**EOF**
